### ISSUE TITLE
Problem with Playing PS2

### ISSUE THREAD

**Issue by x1LesHx** [aldostools/webMAN-MOD#321]
after i play a ps2 game and choose another it say's not ready to fix that i need to restart is there a solution? or the problem common?

**Comment by x1LesHx** [comment]
if there any fix or anything i tried to reinstall rebug i tried to reinstall webman but nothing fix it please there are a fix?

**Comment by x1LesHx** [comment]
i think the solution is to get a fresh hdd and a fresh CFW Thank YOU BTW this is useful plugin <3

**Comment by x1LesHx** [comment]
for that solution i tried that with fresh hdd its the same problem

**Comment by aldostools** [comment]
I haven't been able to reproduce this error.

You could try disabling the content scanning during start up in /setup.ps3

**Comment by x1LesHx** [comment]
same issue

**Comment by gosseux** [comment]
did you try to unmount the first iso before mounting the second (pressing Select +O on your controller)

**Comment by J013k** [comment]
From what I remember after you exit PS2 game, disc should be automatically unmounted.
At least for PS2 titles. PS1 game should be mounted even after exiting the game.

Anyway I have tried to
mount PS2 game -> Exit from this game -> mount another PS2 -> mount different PS2 game -> exit...
But I also couldn't reproduce this error.

Tested on CECH-2004A with Rebug 4.85.1 LITE, webMAN MOD 1.47.26.

What is your HDD capacity?
If it has 2 TiB, well... mostly these HDDs are problematic with PS3.

**Comment by x1LesHx** [comment]
my hdd is 160 gb i tried even the new update the strangest thing is when i tried sMan and its worked flawlessly

**Comment by x1LesHx** [comment]
i think i will stick with sman till i find a fix im so sad cause i love the roms launcher addon

**Comment by J013k** [comment]
Did you tried any other version?
E.g. webMAN MOD 1.47.22 or new one webMAN MOD 1.47.27?
[https://github.com/aldostools/webMAN-MOD/releases](https://github.com/aldostools/webMAN-MOD/releases).

On what CFW you're correctly on?
FERROX, Rebug, SPY...  CEX, DEX, REX... Cobra v8.01, 8.2?
Did you tried to update FW e.g. from 4.85 to 4.86?

**Comment by x1LesHx** [comment]
Yes i try from 1.45 to latest i tried REBUG And Ferrox with cobra 8.1 and 8.2

**Comment by Joonie86** [comment]
No further feedback
